src/proto/string/unicode_collation.rs

```diff
@@ -0,0 +1,182 @@
+//! The Unicode Collation Protocol
+//!
+//! Used in the boot services environment to perform
+//! lexical comparison functions on Unicode strings for given languages
```

Comment:
We do not have a standard yet, but I'd prefer for new code/doc to have proper paragraphs and punctuation.